examples/llm/src/models/layers/attention.py

```diff
@@ -122,16 +122,18 @@ def __init__(self, cfg: DictConfig, device: Optional[str] = None):
 
         self.clip_qkv = cfg.get('attn_clip_qkv')
         self.attn_qk_ln = cfg.get('attn_qk_ln')
+        self.softmax_scale = cfg.get('softmax_scale')
         self.d_model = cfg.d_model
         self.n_heads = cfg.n_heads
 
-        if self.attn_qk_ln or self.clip_qkv:
+        if self.attn_qk_ln or self.clip_qkv or self.softmax_scale:
             self.W_qkv = nn.Linear(self.d_model,
                                    3 * self.d_model,
                                    bias=True,
                                    device=device)
             self.inner_attn = FlashAttention(attention_dropout=cfg.attn_pdrop,
```

Comment:
`1/srqt(d/n_heads)` is standard.
`1/(d/n_heads)` is the recommended muP mod.
Since its part of the model config, it'll get dumped into wandb if we ever need to check what we used.